packages/tables/src/Columns/Concerns/CanBeSelectable.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Filament\Tables\Columns\Concerns;
+
+use Illuminate\Support\Str;
+
+trait CanBeSelectable
+{
+    protected bool $isSelectable = false;
+
+    public function selectable(bool | array $condition = true): static
+    {
+        if (is_array($condition)) {
```

Comment:
This can be a ternary instead of if/else blocks

Reply:
Can we also accept a `Closure`?

Reply:
This method doesn't exist on the column anymore